cmake/modules/CMakeFunctions.cmake

```diff
@@ -51,31 +82,49 @@ function(apply_patches repo_dir patches_dirs base_revision target_branch ret)
         list(APPEND patches ${patches_in_dir})
     endforeach()
     if(NOT patches)
-        message(STATUS "No patches in ${patches_dir}")
+        message(STATUS "[OPENCL-CLANG] No patches in ${patches_dir}")
         return()
     endif()
 
-    message(STATUS "${repo_dir}:")
+    message(STATUS "[OPENCL-CLANG] Patching repository ${repo_dir}")
     # Check if the target branch already exists
     execute_process(
         COMMAND ${GIT_EXECUTABLE} rev-parse --verify --no-revs -q ${target_branch}
         WORKING_DIRECTORY ${repo_dir}
         RESULT_VARIABLE patches_needed
-    )
+	)
     if(patches_needed EQUAL 128) # not a git repo
-        set(ret_not_git_repo 1)
+      set(ret_not_git_repo 1)
```

Comment:
this function had a indent of 4 spaces. Suggest to keep consistent, indent either all 2 or all 4.